GitDiffMargin.Commands/CommandHandlerTextViewCreationListener.cs

```diff
@@ -1,22 +1,23 @@
-﻿namespace GitDiffMargin.Commands
-{
-    using System.ComponentModel.Composition;
-    using Microsoft.VisualStudio.Editor;
-    using Microsoft.VisualStudio.Shell;
-    using Microsoft.VisualStudio.Text.Editor;
-    using Microsoft.VisualStudio.TextManager.Interop;
-    using Microsoft.VisualStudio.Utilities;
+﻿using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.TextManager.Interop;
+using Microsoft.VisualStudio.Utilities;
 
+namespace GitDiffMargin.Commands
+{
     [Export(typeof(IVsTextViewCreationListener))]
     [ContentType("text")]
     [TextViewRole(PredefinedTextViewRoles.Editable)]
     internal class CommandHandlerTextViewCreationListener : IVsTextViewCreationListener
     {
-        private readonly SVsServiceProvider _serviceProvider;
         private readonly IVsEditorAdaptersFactoryService _editorAdaptersFactoryService;
+        private readonly SVsServiceProvider _serviceProvider;
```

Comment:
❔  Why was this reordered? Previously the fields, the constructor parameters, and the assignments in the constructor all appeared in the same order. Now they are inconsistent.